pylarky/starlarky.py

```diff
@@ -0,0 +1,30 @@
+import tempfile
+import subprocess
+
+RUNNER_EXECUATBLE = 'larky-runner'
+INPUT_PARAM = '-input'
+OUTPUT_PARAM = '-output'
+
+
+def evaluate(script, input_data: str) -> str:
```

Comment:
If we don't close (destroy) these files here they will live until a client app (Satellite) is existed which is not good.